src/annotations/actions/thunks.ts

```diff
@@ -2,17 +2,19 @@ import {
   deleteAnnotation,
   getAnnotations,
   getAnnotationStreams,
+  updateAnnotation,
   writeAnnotation,
 } from 'src/annotations/api'
 import {Dispatch} from 'react'
-import {deleteAnnotation as deleteAnnotationAction} from 'src/annotations/actions/creators'
 import {
   setAnnotations,
   setAnnotationStreams,
   Action as AnnotationAction,
+  editAnnotation as patchAnnotation,
```

Comment:
the lack of consistency between these two aliases is kind of confusing, since one is called `patchAnnotation` which is kind of an alias for edit, and one is called `deleteAnnotationAction`. it's not at all clear that they're the same thing later in the code.

just make this `editAnnotationAction` for consistency's sake